packer-build/ubuntu1804.json

```diff
@@ -25,10 +25,10 @@
       ],
       "headless": true,
       "http_directory": "http",
-      "iso_checksum": "e2ecdace33c939527cbc9e8d23576381c493b071107207d2040af72595f8990b",
+      "iso_checksum": "3756b3201007a88da35ee0957fbe6666c495fb3d8ef2e851ed2bd1115dc36446",
```

Comment:
I might be mistaken but the sha256 that I found in the [SHA256SUMS](http://cdimage.ubuntu.com/releases/18.04/release/SHA256SUMS) file for `ubuntu-18.04.5-server-amd64.iso` is: `8c5fc24894394035402f66f3824beb7234b757dd2b5531379cb310cedfdf0996`